python-frontend/src/main/java/org/sonar/python/types/FlowSensitiveTypeInference.java

```diff
@@ -133,6 +133,7 @@ public void visitName(Name name) {
 
       @Override
       public void visitFunctionDef(FunctionDef pyFunctionDefTree) {
+        scan(pyFunctionDefTree.decorators());
```

Comment:
I'm not sure why we do this change. If we want to avoid FNs or FPs, I think that could be reflected with at least one test case in a rule test.

Reply:
After discussion, I will drop this commit as we already use the heuristic to look if `flask` is imported in a file.